vagrant-build/Vagrantfile

```diff
@@ -0,0 +1,21 @@
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/bionic64"
+
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 2048
+  end
+
+  config.vagrant.plugins = "vagrant-disksize"
+  config.disksize.size = "20GB"
+
+  config.vm.provision "shell", inline: $shell, privileged: false
+
+  config.ssh.extra_args = ["-t", "cd SkiffOS; bash -l"]
+end
+
+$shell = <<CONTENTS
+sudo apt update
+sudo apt install -y cmake g++ make mtools python unzip
+
+git clone --recursive https://github.com/paralin/SkiffOS.git
```

Comment:
The files should be mounted directly from the parent filesystem somehow with a bind-mount and/or filesystem share, if this is not possible, they should be copied in from the Skiff repository (ignoring files not tracked in Git) rather than cloning the repo from GitHub.

The reason is that the master branch might be at a different position than the person running the repo expects.